pkg/manifests/nmstateconfig.go

```diff
@@ -3,15 +3,32 @@ package manifests
 import (
 	"context"
 	"fmt"
+	"os"
 	"reflect"
 
 	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
 	"github.com/openshift/assisted-service/models"
 	"github.com/openshift/assisted-service/pkg/staticnetworkconfig"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/yaml"
 )
 
+type NMStateConfig struct {
+	Interfaces []struct {
+		IPV4 struct {
+			Address []struct {
+				IP string `yaml:"ip,omitempty"`
+			} `yaml:"address,omitempty"`
+		} `yaml:"ipv4,omitempty"`
+		IPV6 struct {
+			Address []struct {
+				IP string `yaml:"ip,omitempty"`
+			} `yaml:"address,omitempty"`
+		} `yaml:"ipv6,omitempty"`
+	} `yaml:"interfaces,omitempty"`
+}
+
 func getNMStateConfig() (aiv1beta1.NMStateConfig, error) {
 	var nmStateConfig aiv1beta1.NMStateConfig
 	if err := GetFileData("nmstateconfig.yaml", &nmStateConfig); err != nil {
```

Comment:
Now that we are requiring an nmstateconfig.yaml to get Node0 can you exit here with error if file does not exist.